client/elements/sc-page-search.html

```diff
@@ -592,9 +588,41 @@ <h2 class="search-result-title">[[_calculateTitle(item)]]</h2>
               if (!item.heading.title) {
                   return '';
               } else {
-                  return item.acronym ? item.acronym : item.uid;
+                  return item.acronym ? `${this._convertAcronym(item.acronym)} — ${item.author}` : `${this._transformId(item.uid)} — ${item.author}`;
+              }
+          }
+
+          _convertAcronym(acronym) {
+              if (acronym.match(/\/\//)) {
+                  return acronym.replace(/\/\//, " (") + ")";
+              } else {
+                  return acronym;
+              }
+          }
+
+          _transformId(rootId) {
+              if (!rootId || !this.expansionReturns) {
+                  return '';
               }
-              return item.heading.title ? item.uid : '';
+              const expansionData = this.expansionReturns;
+              let scAcronym = '';
+              const uidParts = rootId.split('-');
+              let tail = '';
+
+              uidParts.forEach(item => {
+                  if (!expansionData[0][item]) {
```

Comment:
What will happen when `expansionData[0]` is `undefined` or an empty object (`{}`)?

Reply:
I've added a try/catch statement just in case but if the expansiondata is not loaded, it should not get this far.